content/build/cache/backends/registry.md

```diff
@@ -39,7 +39,6 @@ The following table describes the available CSV parameters that you can pass to
 | Name                | Option                  | Type                    | Default | Description                                                          |
 | ------------------- | ----------------------- | ----------------------- | ------- | -------------------------------------------------------------------- |
 | `ref`               | `cache-to`,`cache-from` | String                  |         | Full name of the cache image to import.                              |
-| `dest`              | `cache-to`              | String                  |         | Path of the local directory where cache gets exported to.            |
```

Comment:
I just saw this in passing, and I don't think there is such an option for the `registry` cache exporter, so I removed it.